frappe/public/js/frappe/views/factory.js

```diff
@@ -34,15 +35,17 @@ frappe.views.Factory = class Factory {
 }
 
 frappe.make_page = function(double_column, page_name) {
-	if(!page_name) {
-		var page_name = frappe.get_route_str();
+	if (!page_name) {
+		page_name = frappe.get_route_str();
```

Comment:
```suggestion
		page_name = page_name || frappe.get_route_str();
```

Reply:
Isn't this being handled in the if condition above?

Reply:
If this is just a style thing, I kind of like the current design. I would prefer an if condition over re-assigning a variable to itself.

Reply:
Functionally there is no difference, I usually avoid using if-clause. Same on line https://github.com/frappe/frappe/pull/16199/files#diff-8340e1eaf0f8af672975ed973a816d04663a150aeb9e880884a8889b8533d971L46.

Maybe we will have to decide on one way to maintain the codebase. @surajshetty3416 ?

Reply:
You do you @shariquerik 😄 

Functionally, it does not matter that much:

![image](https://user-images.githubusercontent.com/16315650/158135188-ca5e6c3f-6905-4671-a8a0-97194da38bfe.png)

It does impact [readability](https://www.codereadability.com/javascript-default-parameters-with-or-operator/).

And in any case, this should not be a reason to stall a review. I have only removed the `var` keyword here.